aws_config_stream/release.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
<!--  Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🥜 nitpick:** ‏Better to rely on the `bash` active binary rather than the hardcoded path (on macos `/bin/bash` is bash3, which is very old)

```suggestion
#!/usr/bin/env bash
```